src/App.tsx

```diff
@@ -1,155 +1,82 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from 'react';
+import { Todo } from './types/Todo';
+import { Error, Filter } from './utils/Enum';
+import { getTodos } from './api/todos';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoHeder } from './components/TodoHeder';
+import { TodoList } from './components/TodoList';
+import { TodoError } from './components/TodoError';
 
-const USER_ID = 0;
+const USER_ID = 11211;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState(Error.NOTHING);
+  const [filterTodo, setFilterTodo] = useState(Filter.ALL);
+  const [isActive, setIsActive] = useState(false);
+  const [isComplited, setIsComplited] = useState(false);
+
+  const filteredTodos = useMemo(() => {
+    switch (filterTodo) {
+      case Filter.COMPLETED:
+        return todos.filter(todo => todo.completed);
+
+      case Filter.ACTIVE:
+        return todos.filter(todo => !todo.completed);
+
+      default:
+        return todos;
+    }
+  }, [filterTodo, todos]);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((data) => setTodos(data))
+      .catch(() => setHasError(Error.FETCH));
+  }, []);
+
+  useEffect(() => {
+    if (!todos.find(todo => !todo.completed) && todos.length) {
+      setIsActive(true);
+    } else {
+      setIsActive(false);
+    }
+
+    if (todos.find(todo => todo.completed)) {
+      setIsComplited(true);
+    }
+  }, [todos]);
 
   return (
     <div className="todoapp">
       <h1 className="todoapp__title">todos</h1>
 
       <div className="todoapp__content">
-        <header className="todoapp__header">
-          {/* this buttons is active only if there are some active todos */}
-          <button type="button" className="todoapp__toggle-all active" />
+        <TodoHeder
+          activeButton={isActive}
+        />
+
+        <TodoList
+          filteredTodos={filteredTodos}
+        />
```

Comment:
```suggestion
        <TodoHeder activeButton={isActive} />

        <TodoList filteredTodos={filteredTodos} />
```
There is no need to split the lines if there are < 3 props and the line is not too long